ci/test/03_test_script.sh

```diff
@@ -201,14 +201,30 @@ if [ "${RUN_TIDY}" = "true" ]; then
     false
   fi
 
+  # TODO: Consider enforcing IWYU across the entire codebase.
+  FILES_WITH_FORCED_IWYU="/src/(crypto|index)/.*\\.cpp"
+  jq --arg patterns "$FILES_WITH_FORCED_IWYU" 'map(select(.file | test($patterns)))' "${BASE_BUILD_DIR}/compile_commands.json" > "${BASE_BUILD_DIR}/compile_commands_iwyu_errors.json"
```

Comment:
In commit "ci, iwyu: Treat warnings as errors for specific directories" (1d49346465de8edec85bb2d498859b4fbc346935)

I think it'd be a little better to called this ENFORCED_IWYU instead of FORCED_IWYU. Both make sense but former seems a little clearer.